guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -18,14 +18,84 @@ You can list the available ece-tools commands using:
+   -  [Enable on-demand static content generation]({{ page.baseurl }}/cloud/env/variables-intro.html#scd_on_demand): `SCD_ON_DEMAND=true`
+
+-  **Docker Compose for Cloud**—Improved the [Docker configuration]({{ page.baseurl }}/cloud/reference/docker-config.html) process to launch a {{site.data.var.ece}} development environment in your local workspace. (Requires `ece-tools` version 2002.0.13 or later).
+
+   -  JIRA--MAGECLOUD-2359-->Added a command—`docker:config:convert` that converts PHP configuration files to Docker ENV files.
+
+   -  JIRA--MAGECLOUD--2357-->Split the {{site.data.var.ee}} installation process into separate build and deploy phases so that {{site.data.var.ece}} deploys to a read-only file system. During the build phase, the file system is writeable.
+
+-  JIRA--MAGECLOUD-2205-->**Improved {{site.data.var.ece}} support for third-party extensions**—Downgraded the minimum version requirement for the guzzlehttp/guzzle package in the {{site.data.var.ece}} [composer.json file](({{ page.baseurl }}cloud//reference/cloud-composer.html) to version 6.2 so that the `ece-tools` package is compatible with more extensions.
+
+- JIRA--MAGECLOUD-2363-->**Apply custom changes to your {{site.data.var.ece}} application during the build phase**—We split the build phase into two separate processes so that you can apply custom changes to the generated static content before packaging the application for deployment. The *build:generate* process generates code, applies patches, and generates static content. The *build:transfer* phase deploys the application. See [Application hooks]({{ page.baseurl }}//cloud/project/project-conf-files_magento-app.html#hooks).
+
+-  JIRA--MAGECLOUD-2445--> **Cron scheduling improvements**—Improved the cron job management process during the deploy phase to prevent database locks and other critical issues. Now, all cron jobs stop during the deploy phase and restart after the deploy phase completes.
+
+-  **Environment configuration checks**—Improved validation of the environment configuration to proactively warn customers about version incompatibilities and configuration errors before building and deploying {{site.data.var.ece}}.
+
+   -  JIRA--MAGECLOUD-2183-->Added version-specific validation to identify unsupported or deprecated {{site.data.var.ece}} environment variables
+   and values.
+
+   -  JIRA--MAGECLOUD-2389-->Added an Elasticsearch compatibility check. Now, the deployment fails if the `.magento/services.yaml` file specifies an Elasticsearch version that is incompatible with the elasticsearch composer package used to install {{site.data.var.ee}} v2.1.4 and later. Previously, you could deploy with an unsupported version of the Elasticsearch service, which caused product catalog issues after site deployment. See [Setup Elasticsearch]({{ page.baseurl }}/cloud/project/project-conf-files_services-elastic.html).
```

Comment:
There is one Elasticsearch instance that is not caps; is that on purpose?

Reply:
Yes:  "The elasticsearch composer package -  https://packagist.org/packages/elasticsearch/elasticsearch

To communicate with elasticsearch magento is using the package elasticsearch/elasticsearch. (See [wiki](https://wiki.corp.magento.com/display/MCP/Thunder+Team+-+ece-tools+validation+of+elasticsearch+compatibility) --- Technically, it's the `elasticsearch/elasticsearch` composer package.  --